Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -322,6 +322,21 @@ TEST_F(SerializerTest, BadStringValue) {
       Status(FirestoreErrorCode::InvalidArgument, "ignored"), bytes);
 }
 
+TEST_F(SerializerTest, BadStringValue2) {
+  std::vector<uint8_t> bytes;
+  Status status =
+      serializer.EncodeFieldValue(FieldValue::StringValue("a"), &bytes);
+  EXPECT_TRUE(status.ok());
```

Comment:
If we're going to have a lot of lines like this in our tests I think we should probably have an ASSERT_OK(status) that checks `ok()` but then fails with the message if not ok.

See the immutable SortedMap tests for examples of how to define assertions that show more than just the condition:

Definition:
https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/test/firebase/firestore/immutable/testing.h#L49

Usage:
https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/test/firebase/firestore/immutable/sorted_map_test.cc#L80

Once you have a function that returns a `testing::AssertionResult` you can wrap up the call like so:

```c++
#define ASSERT_OK(status) ASSERT_TRUE(StatusOk(status))
```

Reply:
Done. (Also added ASSERT_NOT_OK and EXPECT_*)